app/models/book.rb

```diff
@@ -1,3 +1,5 @@
 class Book < ApplicationRecord
+  has_many :rents, dependent: :nullify
```

Comment:
Ok, just to know: why did you choose `nullify` option?

Reply:
I choose this option because it seems to be the less dangerous option, because it will allow us delete a book even if it has been rented sometime but will not delete all the rents. 
Maybe with more knowledge about the application domain we can choose another option like restrict_with_error or destroy.